static/src/javascripts/projects/commercial/modules/messenger.js

```diff
@@ -186,7 +186,7 @@ const off = (window: WindowProxy): void => {
 
 export const register = (
     type: string,
-    callback: () => void,
+    callback: () => any,
```

Comment:
Think this is unavoidable due to `setBackground` (one of the potential callbacks) either returning null or a promise.